chapters/welcome.ipynb

```diff
@@ -37,24 +37,36 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
    "metadata": {
     "scrolled": true
    },
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Solr is up and responding.\n",
-      "Zookeeper is up and responding.\n",
-      "\n",
-      "All Systems are ready. Happy Searching!\n"
+     "ename": "PermissionError",
```

Comment:
@dcrouch26 We thought that moving welcome.ipynb into `chapters/` would fix this, but it's still one directory too high. Ultimately, the code to load system.config needs to be improved because it's breaking in multiple places (currently at least here and chapter 7 inside the webserver).

Reply:
I went ahead and fixed the `system.config` issues by loading relative to the `environment.py` file instead of the calling context. Should be all set now.